client/src/slices/orphaned-record.slice.ts

```diff
@@ -0,0 +1,82 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { OrphanedRecordClient } from '../client/api';
+import { ApiOrphanedRecord } from '../models/api-response';
+
+export interface OrphanedRecordState {
+  rows: ApiOrphanedRecord[],
+  totalRowsCount: number
+  count: number
+  countLoading: boolean
+  pageLoading: boolean
+  lastUpdated: number
+}
+
+export const orphanedRecordInitialState: OrphanedRecordState = {
+  rows: [],
+  totalRowsCount: 0,
+  count: 0,
+  countLoading: false,
+  pageLoading: false,
+  lastUpdated: 0,
+};
+
+const name = 'orphanedRecord';
+
+export const fetchCount = createAsyncThunk(
+  `${name}/fetchCount`,
+  async (orgId: number) => {
+    return OrphanedRecordClient.fetchCount(orgId);
+  },
+);
+
+export const fetchPage = createAsyncThunk(
+  `${name}/fetchPage`,
+  async (args: {orgId: number, page: number, limit: number, unit?: string}) => {
+    return OrphanedRecordClient.fetchPage(args.orgId, args.page, args.limit, args.unit);
+  },
+);
+
+export const orphanedSlice = createSlice({
+  name,
+  initialState: orphanedRecordInitialState,
+  reducers: {
+    clear: () => {
+      return { ...orphanedRecordInitialState};
+    },
```

Comment:
I suspect we can get away with just doing this, since redux-toolkit shouldn't be doing any internal mutations.

```suggestion
    clear: () => orphanedRecordInitialState,
```